.php_cs

```diff
@@ -22,25 +22,29 @@ return PhpCsFixer\Config::create()
     ->setRiskyAllowed(true)
     ->setRules([
         '@Symfony' => true,
-        'binary_operator_spaces' => ['align_double_arrow' => true, 'align_equals' => true],
-        'array_syntax' => array('syntax' => 'short'),
-        'blank_line_after_namespace' => true,
-        'header_comment' => array('header' => $header),
-        'lowercase_constants' => true,
-        'lowercase_keywords' => true,
-        'method_separation' => true,
-        'no_closing_tag' => true,
-        'no_extra_consecutive_blank_lines' => [
+        'array_syntax' => [
+            'syntax' => 'short'
+        ],
+        'binary_operator_spaces' => [
+            'align_double_arrow' => true,
+            'align_equals'       => true
+        ],
+        'class_attributes_separation' => [
+            'elements' => [ "const", "method", "property" ]
+        ],
+        'header_comment' => [
+            'header' => $header
+        ],
+        'no_extra_blank_lines' => [
             'tokens' => ['break', 'continue', 'extra', 'return', 'throw', 'use', 'parenthesis_brace_block', 'square_brace_block', 'curly_brace_block']
         ],
-        'no_short_echo_tag' => true,
```

Comment:
keep this one

Reply:
Oh, I see, this isn't a default in the latest version any more!